international/russian/tokenomics.md

```diff
@@ -0,0 +1,115 @@
+
+## Стратегические партнеры Angle и гранты для сообщества
+
+12% первоначального количества ANGLE будет находиться в распоряжении основной команды и предназначаться для выплат сообществу в виде грантов или баг-баунти (bug bounties), стратегическим партнерам, таким как биржи - для листинга, а также наиболее активным и полезным членам сообщества, наравне с консультантами, помогающими развитию протокола.
+
+Хотя программа майнинга ликвидности справедлива в смысле пропорционального распределения, она отдает преимущество пользователям с наибольшим капиталом. Мы считаем, что через гранты и вознаграждения членам сообщества мы сможем распределить и сохранить управление в руках пользователей, независимо от их капитала и, следовательно, способствовать созданию надежного и основанного на здоровой мотивации сообщества.
+
+Часть этих токенов также может быть возвращена обратно в казначейство DAO, в программы майнинга ликвидности или механизм bonding curve, если это будет признано лучшим способом поощрения сообщества.
+
+## Применение токенов
+
+Хотя распределение токенов важно, гораздо важнее, для чего эти токены будут использоваться. Токен ANGLE является краеугольным камнем протокола Angle. Токен дает право участвовать в голосовании за управление с Angle DAO. Аngle DAO будет отвечать за настройку параметров, добавление новых стейблкоинов, утверждение новых типов обеспечения для уже имеющихся стейблкоинов, а также обновления протокола и различные интеграции.
+
+DAO также будет отвечать за эффективное использование залогового обеспечения в протоколе. Протокол владеет некоторым обеспечением, полученным при выпуске пользователями стейблкоинов, а также от Стандартных провайдеров ликвидности (SLPs) и Хеджирующих агентов (HAs): DAO будет субъектом, управляющим стратегиями получения дохода от залогового обеспечения и выбирающим, в каких пропорциях распределять эту доходность между Стандартными провайдерами ликвидности (SLPs).
+
+Кроме того, протокол будет естественным образом накапливать некоторое избыточное обеспечение от транзакционных сборов, процентной доходности и повышения стоимости залога. DAO также будет ответственен за принятие решения о том, как распоряжаться данным избыточным обеспечением.
+
+В то время как часть избыточного обеспечения будет играть роль буфера для поглощения волатильности залогового обеспечения и роста доходности, которые будут возрастать с течением времени, другая часть также может быть выделена для поддержки инициатив касаемо протокола или для выкупа ANGLE в форме аукционов, как это делает Maker DAO. Преимущество обратного выкупа заключается в том, что это выравнивает стимулы держателей ANGLE с остальной частью экосистемы ANGLE. Выкупленные токены ANGLE могут быть либо сожжены, либо возвращены в казначейство DAO, либо распределены через программу майнинга ликвидности.
+
+# Выводы
```

Comment:
Had not paid attention, we need to be wary of the titles: there can only be one paragraph with just one # per article and it needs to be the title
So let's put two ## here